Documentation/security/restrict-pod-access.rst

```diff
@@ -0,0 +1,52 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _security_restrict_pod_access:
+
+========================================
+Restricting privileged Cilium pod access
+========================================
+
+This page shows you how to restrict privileged access to Cilium pods by limiting access from the Kubernetes API, specifically from `kubernetes exec pod`_.
+
+.. include:: gsg_requirements.rst
+
+Background
+----------
+
+The Cilium agent needs some specific Linux capabilities to perform essential system and network operations.
+
+Cilium relies on Kubernetes and Containers to mount filesystems and the environment. Cilium doesn't perform any extra operations that could result in an unsafe volume mount. 
+
+Cilium needs kernel interfaces to properly configure the environment. Some kernel interfaces are part of the ``/proc`` filesystem, which includes host and machine configurations that can't be virtualized or namespaced.
+
+If ``pod exec`` operations aren't restricted, then remote `exec into pods`_ and containers defeats Linux namespace restrictions.
+
+Entering a container that runs with ``X`` privileges like Cilium requires the ``CAP_SYS_ADMIN ``capability in your current user namespace and in the target Cilium user namespace. Because Cilium runs in the initial user namespace, entering its namespace requires ``CAP_SYS_ADMIN`` capability, which already provides privileged access.
```

Comment:
Formatting typo: `the ``CAP_SYS_ADMIN ``capability` instead of `the ``CAP_SYS_ADMIN`` capability`